libmproxy/script/reloader.py

```diff
@@ -1,6 +1,12 @@
 import os
+import fnmatch
+import sys
 from watchdog.events import PatternMatchingEventHandler
-from watchdog.observers import Observer
+if sys.platform == 'darwin':
+    from watchdog.observers.polling import PollingObserver as Observer
+else:
+    from watchdog.observers import Observer
+# Use PollingObserver on OS X, and natvie Observer on Linux and Windows
```

Comment:
This comment is not very helpful for the reader. How about "The OSX reloader is watchdog 0.8.3 breaks when unobserving paths. We use the PollingObserver instead."?